### PR TITLE
Minor fixups

### PR DESCRIPTION
**Pull request by Nekotekina**

### CODE REVIEW

File: rpcs3/rpcs3qt/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,59 @@
-file(GLOB SRC_FILES "*.cpp")
-file(GLOB UI_FILES "*.ui")
+set(SRC_FILES
+	about_dialog.cpp
+	auto_pause_settings_dialog.cpp
+	breakpoint_handler.cpp
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/CMakeLists.txt]: This really looks like a pain in the ass to manage.
Does VS do this automatically?

Comment (Nekotekina) [reply rpcs3/rpcs3qt/CMakeLists.txt]: I dunno, you just add or remove line with source name.

Comment (Megamouse) [reply rpcs3/rpcs3qt/CMakeLists.txt]: Just wondering. What's the reason for this change? Was the old implementation faulty?

Comment (Nekotekina) [reply rpcs3/rpcs3qt/CMakeLists.txt]: GLOB with *.cpp is one of the possible culprits for unnecessary full rebuilds. Keeping file list is just a recommended way of doing this. Also it makes sure that reconfiguration happens on rebuild.

**Pull request merged**